synapse/handlers/auth.py

```diff
@@ -119,6 +110,14 @@ def __init__(self, hs):
 
         self._clock = self.hs.get_clock()
 
+        # Expire old UI auth sessions after a period of time.
+        self._clock.looping_call(
+            lambda: run_as_background_process(
+                "expire_old_sessions", self.expire_old_sessions
+            ),
+            5 * 60 * 1000,
+        )
```

Comment:
`looping_call` lets you pass params to the callee, so this can be written:

```suggestion
        self._clock.looping_call(
            run_as_background_process
            5 * 60 * 1000,
            "expire_old_sessions",
            self.expire_old_sessions,
        )
```

(it may or may not be clearer, but it's what we do elsewhere, and consistency ftw)

also: remember to stop this running when we're not on the master!